src/js/core/util/rotaryScrolling.js

```diff
@@ -39,18 +39,41 @@
 				 * Scroll step
 				 * @type {number}
 				 */
-				scrollStep = 40;
+				scrollStep = 40,
+				/**
+				 * Time after bound effect will be hidden.
+				 * type {number}
+				 */
+				bouncingTimeout = 1000;
```

Comment:
I think this is const value. You can change this to 'BOUNCING_TIMEOUT'

Reply:
Done.